### PR TITLE
fix(router-lite): location URL with no history and fragment routing

### LINKED ISSUE #2003: Router-lite duplicates url on page reload
This is probably a weird use-case. I am trying to incorporate an Aurelia app into a Wordpress admin page.

More info here: https://discourse.aurelia.io/t/integrating-aurelia-with-wordpress-backend/5474

Seem to be working, however when I reload the page. The route is duplicated within the url bar of the browser.

e.g. on first load:

`http://localhost:8888/wp-admin/admin.php?page=page-01/#/database?page=page-01`

`/#/database?page=page-01` is added by the `router-lite`. Then when the page is reloaded the new url is: `http://localhost:8888/wp-admin/admin.php?page=page-01%2F#/database?page=page-01/#/database?page=page-01`

I have tried to play around with the `basePath` config option, but have not had any success. If I set the basePath to this:
```
basePath: '/wp-admin'
```

I get this error

```
Error: AUR3501: Unexpected '.php' at index 6 of '/admin.php'
```

Any suggestions?

Is is possible to use the route with a type of in-memory url system rather then modifying the main url of the page?

### PR DESCRIPTION
**Pull request by Sayan751**
# Pull Request

## 📖 Description

Even with historyStrategy set to `none`, the router-lite interacted with the browser history for the first time when the app started. This caused the issue reported in #2003. This PR avoids that.

### 🎫 Issues

Closes #2003.

## ⏭ Next Steps

<br/>
<br/>

cc: @ivanbacher

### CODE REVIEW

File: packages/__e2e__/10-router-lite-none-history-with-hash/CHANGELOG.md
Comment (bigopon) [inline packages/__e2e__/10-router-lite-none-history-with-hash/CHANGELOG.md:1]: Maybe delete this file content.

**Review by bigopon — APPROVED** [review]
Nice work thanks @Sayan751 @ivanbacher 
Can you add a few wordss stating the issue being fixed @Sayan751

**Comment by Sayan751** [comment]
@bigopon Updated the description. However, don't merge this yet. Just remembered that I missed adding the new e2e tests in the CI pipeline. I need to do that.

File: packages/__e2e__/10-router-lite-none-history-with-hash/src/router-lite.spec.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import { expect, test } from '@playwright/test';
```
Comment (Sayan751) [inline packages/__e2e__/10-router-lite-none-history-with-hash/src/router-lite.spec.ts:1]: Incorporate the tests with CI.

**Comment by bigopon** [comment]
@Sayan751 I think you forgot to push

File: packages/__e2e__/10-router-lite-none-history-with-hash/CHANGELOG.md
Comment (Sayan751) [reply packages/__e2e__/10-router-lite-none-history-with-hash/CHANGELOG.md:1]: Done.

**Pull request merged**